packages/server-wallet/src/wallet/store.ts

```diff
@@ -650,11 +650,6 @@ export class Store {
         tx
       );
 
-      /** todo:
-       * What happens if Bob is adding his signature to prefund0 from Alice?
```

Comment:
Why is this stale? It seems like it might have manifested in a confusing situation that lead to a DB constraint violation in staging.

<img width="585" alt="Screen Shot 2020-11-01 at 12 36 45 PM" src="https://user-images.githubusercontent.com/8432675/97814830-ea563380-1c3e-11eb-9617-29a72017aec5.png">

Reply:
This method is not called by Bob — it is only ever called inside `wallet.createChannel` or `wallet.createLedgerChannel`